routes/favicon.coffee

```diff
@@ -0,0 +1,27 @@
+request = require 'request'
+fs = require 'fs'
+tmpdir = require 'os'
+	.tmpdir()
+path = require 'path'
+CACHEFILE = 'favicon.ico'
+
+module.exports = (req, res, next) ->
+	res.sendFile CACHEFILE,
+		root: tmpdir
+		(error) ->
+			if error
```

Comment:
Then, notice you're doing a bunch of stuff `if error`, but nothing otherwise. You can skip another level of indentation by doing this the other way around.
```
return unless error
request 'htps://vine.co/', (error, response, body) ->
```